app/src/main/java/com/concordia/mcga/helperClasses/DatabaseHelper.java

```diff
@@ -5,22 +5,21 @@
 import android.database.sqlite.SQLiteException;
 import android.database.sqlite.SQLiteOpenHelper;
 import android.util.Log;
-
+import com.concordia.mcga.exceptions.MCGADatabaseException;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.sql.SQLException;
 
 
 //Part of the code was taken from https://blog.reigndesign.com/blog/using-your-own-sqlite-database-in-android-applications/
 public class DatabaseHelper extends SQLiteOpenHelper {
+
     private static final String DATABASE_NAME = "development.db";
-    private static SQLiteDatabase db;
 
     //The Android's default system path of our application database.
     private static String databasePath;
-    private static Context context;
+    private Context context;
     private static DatabaseHelper databaseHelper;
```

Comment:
Agreed. However, this is because our DatabaseHelper is a singleton instance. The methods in this class are designed to be used by a singleton, to simplify the logic of using it. Removing its 'Singleton'-ness would be a significantly larger task.